### ISSUE TITLE
External_data: requires make to work when testing

### ISSUE THREAD

**Issue by johan-andruejol** [girder/girder#1266]
Hey guys,

I was trying to use External_data in my plugin and I couldn't figure out why it wasn't working on my machine when it ran fine on the dashboard. 
It's because you need to run make in your test directory otherwise the data isn't downloaded.

Right now the documentation doesn't say that you MUST run cmake if you want your tests with External_data to pass. Could we add that into the documentation ?

@jbeezley: What do you think ?

**Comment by jbeezley** [comment]
Yes, running `make` (or `ctest --build-and-test`) is required for both `ExternalData` and to create the virtual environment for the packaging tests.  Looking at the docs [here](http://girder.readthedocs.org/en/latest/development.html#server-side-testing), there is definitely room for clarification.
